### ISSUE TITLE
Results not replicating for Atari Environments

### ISSUE THREAD

**Issue by rach0012** [hill-a/stable-baselines#43]
I tried training different models (ACER and A2C) on a couple of atari games like Space Invaders and Seaquest but the models don't seem to train on them -- they get rewards between 5-10 even after training for 1M+ steps. Has anyone had any success in training on Atari games?  

For reference, I am using the code example that is similar to the atari example in the documentation

**Comment by araffin** [comment]
Hello,

>For reference, I am using the code example that is similar to the atari example in the documentation

RL is very sensitive to change in hyperparameters. The example code just demonstrate the use and does not give the hyper-parameters. I recommend you to use the different `run_atari` scripts (e.g. for A2C https://github.com/hill-a/stable-baselines/blob/master/stable_baselines/a2c/run_atari.py). Even though default hyper-parameters are the one used for atari, the number of workers (i.e., number of environments run in parallel), which is an important one, is missing.

I did train A2C on breakout and obtain good results. You can reproduce the experiment here (google colab notebook): https://colab.research.google.com/drive/14NwwEHwN4hdNgGzzySjxQhEVDff-zr7O


Note: the frame-stacking is missing in ACER run_atari, the PR #45 fixes that (and will be merged soon)

**Comment by araffin** [comment]
@rach0012 
I just trained an A2C agent during 5M on seaquest using this colab: https://colab.research.google.com/drive/1mBDRxZZu0ajS8tiR1DQSWSAqaUSURbvL
You can download the trained agent here: https://drive.google.com/open?id=1fibRLLmN6YO1h3CX8Jtv4QKhOLBrFbw1

The agent achieves a score of ~800 (with stochastic actions) but this different is from the episode reward (which is around 10)

the enjoy script:
```python
from stable_baselines import A2C
from stable_baselines.common.cmd_util import make_atari_env
from stable_baselines.common.vec_env import VecFrameStack
# Create n environments
env = make_atari_env('SeaquestNoFrameskip-v4', num_env=1, seed=0)
# Stack 4 frames
env = VecFrameStack(env, n_stack=4)
# Load the trained agent
model = A2C.load("seaquest_a2c.pkl")

reward_sum = 0
obs = env.reset()
while True:          
    action, _states = model.predict(obs, deterministic=False)
    obs, rewards, dones, info = env.step(action)
    env.render()


```

**Comment by rach0012** [comment]
Great, thank you so much for getting back to me. Just one more question - how did you get the score instead of the episode reward? I believe I never saw the score and just the episode reward which made me believe that training was not happening optimally. Thanks again!

**Comment by rach0012** [comment]
Just to be clear, I am wondering how to obtain the training curve i.e. plot of the score (not episode reward) with respect to the number of timesteps, thanks!

**Comment by araffin** [comment]
>how did you get the score instead of the episode reward?

I understand perfectly your question ;) To be honest, I just looked at the game window. I tried looking at the code (plotting script, benchmark script) but did not find anything yet... 
You can open an issue on baselines repo, they generated the curve so they should know.

**Comment by araffin** [comment]
They answered in the OpenAI repo ;) (see issue #667) : you can use Monitor wrapper for that.
